packages/ckeditor5-media-embed/src/mediaembedediting.js

```diff
@@ -257,7 +257,7 @@ export default class MediaEmbedEditing extends Plugin {
 						const idAndParams = match[ 1 ];
 
 						return (
-							'<div style="position: relative; padding-bottom: 100%; height: 0; padding-bottom: 56.2493%;">' +
+							'<div style="position: relative; height: 431px;">' +
```

Comment:
This line was causing overflow issue, I changed height to 431px to do the same as we do for GoogleContent embed